octoprint_signalclirestapi/__init__.py

```diff
@@ -152,6 +154,10 @@ def print_failed_event(self):
     def print_paused_event(self):
         return self._settings.get_boolean(["printpausedevent"])
     
+    @property
+    def filament_change_event(self):
+        return self._settings.get(["filamentchangeevent"])
```

Comment:
I just tested your changeset and it works really well!

The only small thing I've found during code review is in this line (line 159) here. I think it would be better to use `get_boolean` instead of `get` here. I _think_ this change has no operational impact (at least I couldn't find any), but I think it would be more consistent to use `get_boolean` here.

Apart from that everything looks fine!

Many thanks for this really nice contribution!